src/__tests__/lib/command/select.test.ts

```diff
@@ -21,21 +20,26 @@ describe('select', () => {
 	const singleItemList = [item1]
 	const booleanConfigValueMock = jest.fn()
 
-	const commandWithProfile = (profile: Profile): SmartThingsCommandInterface => ({
-		...buildMockCommand({ output: 'output.yaml' }, profile),
+	const commandWithProfile = (profile: Profile): SmartThingsCommand<SmartThingsCommandFlags> => ({
+		logger: log4js.getLogger('cli'),
+		cliConfig: {
+			profileName: 'default',
+			mergedProfiles: { default: { profile } } as ProfilesByName,
+			profile,
+		} as CLIConfig,
 		booleanConfigValue: booleanConfigValueMock,
-	})
+	} as unknown as SmartThingsCommand<SmartThingsCommandFlags>)
 	const command = commandWithProfile({})
-	const config = {
-		tableFieldDefinitions: [],
+	const config: SelectFromListConfig<typeof item1> = {
 		primaryKeyName: 'str',
 		sortKeyName: 'num',
 	}
 
-	const listItems = jest.fn()
+	const listItems = jest.fn().mockResolvedValue(list)
 	const stringGetIdFromUserMock = jest.mocked(stringGetIdFromUser)
 	const outputListMock = jest.mocked(outputList)
 
+	const consoleLogSpy = jest.spyOn(console, 'log').mockImplementation(() => { /*no-op*/ })
```

Comment:
"no-op" is short for "no operation". It's often used to represent that something doesn't do anything. Here I'm using it to avoid the "empty body" warning and explicitly state that I don't need this function to do anything.